src/services/opcua_binary-services.pac

```diff
@@ -14,128 +14,132 @@ type Service(msg_body: Msg_Body) = record {
     namespace_idx : uint8;
```

Comment:
Most of the changes in this file are merely cleanup. I moved all of the stubbed out/unimplemented services to the bottom below a comment and then alphabetized both the implemented and stubbed services within their respective regions.